build/committee/1345259/contributions/index.json

```diff
@@ -160,13 +174,76 @@
     "Tran_NamF": null,
     "Tran_NamL": "CTA/ABC PAC"
   },
+  {
+    "Filer_ID": "1345259",
+    "Tran_Amt1": 12000.0,
+    "Tran_Date": "2016-09-25",
+    "Tran_NamF": null,
+    "Tran_NamL": "CTA/ABC PAC"
+  },
+  {
+    "Filer_ID": "1345259",
+    "Tran_Amt1": 12000.0,
+    "Tran_Date": "2016-09-25",
+    "Tran_NamF": null,
+    "Tran_NamL": "CTA/ABC PAC"
+  },
+  {
+    "Filer_ID": "1345259",
+    "Tran_Amt1": 12000.0,
+    "Tran_Date": "2016-09-25",
+    "Tran_NamF": null,
+    "Tran_NamL": "CTA/ABC PAC"
+  },
   {
     "Filer_ID": "1345259",
     "Tran_Amt1": 13000.0,
     "Tran_Date": "2016-09-25",
     "Tran_NamF": null,
     "Tran_NamL": "CTA/ABC PAC"
   },
+  {
```

Comment:
It looks really strange that they made 4 idental contributions on the same date.  We ought to ask Suzanne wha she thinks.

Reply:
I looked in the processed data, and it was there... Perplexing indeed, but perhaps not inaccurate.

Reply:
Busy week for me, but I will check it out before Tuesday eve.

Reply:
All right, thanks for the investigation. I'll assume you've got it from here and that this is fine to merge for now.